reducers/chatsReducer.js

```diff
@@ -24,17 +25,19 @@ function chatsReducer(state = chatsInitialState, action) {
                 chats: state.chats.concat(payload)
             };
         case types.CREATE_USER_IN_CHAT:
-            const { cid, userId, displayName } = payload;
+            const { id, display_name: displayName, cid } = payload;
+            const { chats } = state;
             const user = {
```

Comment:
chats doesn't look at all like the format that you proposed earlier and i mentioned in the pr https://github.com/emote-chat/server/pull/21, or am i losing it?